transaction_apple_pay_details_integration_test.go

```diff
@@ -49,6 +49,9 @@ func TestTransactionApplePayDetails(t *testing.T) {
 	if tx.ApplePayDetails.CardholderName == "" {
 		t.Fatal("Expected ApplePayDetails to have CardholderName set")
 	}
+	if tx.ApplePayDetails.ProductID == "" {
```

Comment:
I think this values exists only if certain conditions were met (sending level 2 and 3 data)
So maybe we can mark it as optional?